packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx

```diff
@@ -330,6 +330,14 @@ const TemplateDialog: FC<Props> = (props) => {
   )
 }
 
+const selectTemplate = (template, props, dispatch) => {
+  dispatch(selectTmpl(null))
+  if (!template) return
```

Comment:
I propose to write conditions with curly brackets, even when they're single line